src/graphic/Text.ts

```diff
@@ -501,12 +503,10 @@ class ZRText extends Displayable<TextProps> implements GroupLike {
 
         if (needDrawBg || textPadding) {
             // Consider performance, do not call getTextWidth util necessary.
-            let outerWidth = contentBlock.width;
-            textPadding && (outerWidth += textPadding[1] + textPadding[3]);
-            const boxX = adjustTextX(baseX, outerWidth, textAlign);
+            const boxX = adjustTextX(baseX, contentWidth, textAlign);
             const boxY = adjustTextY(baseY, outerHeight, verticalAlign);
 
-            needDrawBg && this._renderBackground(style, style, boxX, boxY, outerWidth, outerHeight);
+            needDrawBg && this._renderBackground(style, style, boxX, boxY, contentWidth, outerHeight);
```

Comment:
Is it a mistake to render the background with `contentWidth` instead of `outerWidth`, which will include the padding